css/index.css

```
@@ -1,3 +1,457 @@
+
+* {
+  box-sizing: border-box;
+}
+body {
+  display: flex;
+  flex-direction: column;
+  width: 75%;
+  margin: 0 auto;
+}
+header,
+main,
+footer {
+  align-content: center;
+}
+header {
+  position: sticky;
+}
+header,
+nav {
+  display: flex;
+  width: 100%;
+}
+nav ul {
+  display: flex;
+  justify-content: space-around;
+  align-content: center;
+  width: 70%;
+  padding-right: 10%;
+}
+nav ul li {
+  justify-content: flex-start;
+  font-weight: bold;
+  padding: 0;
+  margin: 20px auto;
+}
+nav ul li a {
+  text-decoration: none;
+  color: gray;
+}
+nav > a {
+  padding: 5px 0px;
+}
+.active {
+  color: red;
+}
+nav ul li a:hover {
+  background-color: black;
+  border: 1px solid yellow;
+  border-radius: 25px;
+  padding: 2px;
+  text-decoration: none;
+}
+nav .logo:hover {
+  border-radius: 25px;
+  background-color: yellow;
+}
+#container1 {
+  display: flex;
+  justify-content: space-around;
+  padding: 80px 90px 80px 60px;
+}
+div.top-content-left {
+  width: 50%;
+}
+#container1 > div > h1 {
+  font-size: 70px;
+  text-align: center;
+}
+#start-btn {
+  margin: 10px 35%;
+  color: black;
+  padding: 5px 0;
+  font-weight: bold;
+  border: 1px solid black;
+  width: 40%;
+  height: 12%;
+}
+.top-content-left > h1 {
+  font-family: "Montserrat", Roboto;
+}
+.middle-content {
+  display: flex;
+}
+div.middle-column {
+  width: 50%;
+  margin: 30px 100px 40px 0;
+}
+.middle-img {
+  width: 100%;
+}
+.bottom-content {
+  display: flex;
+}
+div.bottom-column {
+  width: 33%;
+  margin: 30px 60px 30px 0;
+}
+address {
+  padding: 20px;
+}
+#cpyrght {
+  text-align: center;
+}
+/************************************************/
+/*Services.html*/
+section#svcs-top {
+  display: flex;
+  flex-direction: column;
+}
+.services-header-img {
+  width: 100%;
+  border: 2px solid red;
+}
+#svcs-top {
+  display: flex;
+  border-bottom: 2px solid;
+}
+#svcs-top > div {
+  margin: 20px 0px;
+  width: 100%;
+}
+div#svcs-mid {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: space-evenly;
+  align-content: center;
+}
+#svcs-mid > div {
+  width: 35%;
+  height: 250px;
+  border: 1px solid;
+  padding: 10px;
+  margin: 15px 0px;
+  display: flex;
+  flex-direction: column;
+}
+#svcs-mid div > button {
+  align-self: flex-start;
+  height: 40px;
+  width: 150px;
+  margin: auto 0;
+  background-color: white;
+  border-style: outset;
+}
+section#bttm-content {
+  /*padding: 20px 15px;*/
+  border-top: 3px solid black;
+  width: 100%;
+}
+section#bttm-content div {
+  margin: 0 auto;
+  width: 48%;
+}
+#bttm-content > div > ul {
+  list-style-type: georgian;
+  list-style-position: inside;
+  list-style: circle;
+}
+#bttm-content > div {
+  display: inline-block;
+  width: 49%;
+  max-width: 100%;
+}
+/*end of services.html styling*/
+/**************************************/
+
+@media (max-width: 470px) {
+  /* start of phone styles */
+  body,
+  header,
+  main {
+    width: 470px;
```

```suggestion
    width: 90%;
```
You should take this width out altogether. That would keep the page size at 75% on every screen size, which is a little tight on the smaller screens. So I would suggest wider than 75%, maybe 90%. The way you have the width hard coded at 470px here on screens from 0-469px the screen size is 470px, so some of it is getting cut off.